bundle-debug-info: document bundle-debug-info tool after refactoring (#110)
bisdn/bisdn.github.io: tools/bundle-debug-info.md

---
title: bundle-debug-info
parent: Tools
---

# bundle-debug-info

The bundle-debug-info script enables customers to create a tar file with the current switch state. It gathers information like port status, system logs and configuration, to ease debugging and reporting errors on the switch platform. To execute properly, please run the following command on the switch with root privileges.

```
bundle-debug-info
```

> **Note**: In all BISDN Linux release prior 4.0 the `bundle-debug-info` tool is called `basebox-support` and has a slightly decreased functionality.

Reviewer: In BISDN Linux releases prior to 4.0, the `bundle-debug-info` tool was called `basebox-support` and lacked some of the functionality of the new tool.

> **Note**: In all BISDN Linux release prior 4.0 the `bundle-debug-info` tool is called `basebox-support` and has a slightly decreased functionality.

Reviewer: ```suggestion
**Note**: In BISDN Linux releases prior to 4.0, the `bundle-debug-info` tool was called `basebox-support` and lacked some of the functionality of the new tool.
```
